biz.aQute.bndlib/src/aQute/bnd/osgi/Analyzer.java

```diff
@@ -648,11 +648,15 @@ void doPlugins() {
 		for (AnalyzerPlugin plugin : getPlugins(AnalyzerPlugin.class)) {
 			try {
 				Processor previous = beginHandleErrors(plugin.toString());
-				boolean reanalyze = plugin.analyzeJar(this);
-				endHandleErrors(previous);
-				if (reanalyze) {
-					classspace.clear();
-					analyzeBundleClasspath();
+				try {
+					boolean reanalyze = plugin.analyzeJar(this);
+					if (reanalyze) {
```

Comment:
Yes! (not sure why you need to set reanalyze to false, perhaps the compiler whines...)